fares-data-build-tool/create-data
# fix: Footer including govuk crest

## Description

Alterations to footer to include govuk crest and OGL text.

## Testing instructions

Run locally and see new footer. Needs checking on each environment to see if the assets are being accessed properly.
